php/libraries/NDB_Page.class.inc

```diff
@@ -280,6 +280,27 @@ class NDB_Page
     }
 
     /**
+     * Creates an HTML checkbox and adds it to the current form.
+     *
+     * @param string $field   The fieldname for this checkbox
+     * @param string $label   The label to attach to this checkbox
+     * @param string $options Options to pass to HTML_QuickForm
+     * @param string $closer  ?????
+     *
+     * @return none
+     */
+    function addCheckbox($field, $label, $options=array(), $closer='</label>')
```

Comment:
not sure what $closer does but this is present in the main code base so i left it in